src/Counter.elm

```diff
@@ -15,21 +15,21 @@ type Msg
 
 init : Model
 init =
-    Model 0
+    0
 
 
 update : Msg -> Model -> Model
 update msg model =
     case msg of
         Plus ->
-            { model | counter = model.counter + 1 }
+            model + 1
 
         Minus ->
-            if model.counter == 0 then
+            if model == 0 then
                 model
```

Comment:
I'd name this module `NonNegativeCounter`

Reply:
we prefer `Counter`